src/app/pages/charts/charts.routing.ts

```diff
@@ -2,14 +2,16 @@ import { Routes, RouterModule }  from '@angular/router';
 
 import { Charts } from './charts.component';
 import { ChartistJs } from './components/chartistJs/chartistJs.component';
+import {MorrisComponent} from "./components/morrisJs/morris.componenet";
```

Comment:
Please make sure you used correct style guide (spaces around the component name).
Single quotes should be used.